src/main/java/edu/kit/compiler/codegen/pattern/UnknownPattern.java

```diff
@@ -22,10 +21,15 @@
 
```

Comment:
``except with Mode `b` ``

This should probably be removed, since `Unknown` nodes with Mode `b` are matched (although the call only returns a placeholder).